tests/CMakeLists.txt

```diff
@@ -292,6 +293,7 @@ if(AVIF_BUILD_APPS)
     if(NOT AVIF_CODEC_AOM_ENABLED OR NOT AVIF_CODEC_AOM_ENCODE)
         # Only aom encoder supports AV1 lossless encoding.
         set_property(TEST test_cmd_animation PROPERTY DISABLED True)
+        set_property(TEST test_cmd_depthext PROPERTY DISABLED True)
```

Comment:
Future,unrelated: The latest SVT-AV1 releases also support lossless encoding now. So we can update the consition and the comment at lines 293-294.